### DISCUSSION TITLE
Z‐Stick Gen5 working on Synology, but Aeotec Z-wave stick doesn't?

### DISCUSSION THREAD

**Discussion by stephanvierkant** [zwave-js/zwave-js-ui#3442] [Q&A] [answered]
I've got a Z-Wave controller working on Synology DS220+. Zwave-js-ui is running on Docker. But now I've bought a newer stick (Aeotec Z-wave stick 7) that doesn't work. I've ran the whole [/sbin/modprobe usbserial thing](https://www.reddit.com/r/synology/comments/ivqbpf/comment/gbyi5oy/?utm_source=share&utm_medium=web2x&context=3) and restarted zwave-js-ui. I've attached both sticks and only the gen5 is working. I've switched ports, but that didn't change a thing. I'm not using using any hubs or extension cables.

The stick does work on my laptop (Ubuntu), so it isn't broken.

Does anybody know why this doesn't work?

Some debug information (ttyACM0 is the old stick):

```
dmesg | grep tty
[31514.010601] cdc_acm 1-1:1.0: ttyACM0: USB ACM device 
[31529.064174] synobios get empty ttyS current 
[31551.787403] cdc_acm 1-1:1.0: ttyACM0: USB ACM device 
[31589.862958] synobios get empty ttyS current 
[31650.617963] synobios get empty ttyS current 
[31711.305980] synobios get empty ttyS current
```
```
lsusb -ciu 
|__usb1          1d6b:0002:0404 09  2.00  480MBit/s 0mA 1IF  (Linux 4.4.302+ xhci-hcd xHCI Host Controller 0000:00:15.0) hub 
|__1-1         0658:0200:0000 02  2.00   12MBit/s 100mA 2IFs ( ffffffd1ffffffb2ffffffdbffffffad) 
1-1:1.0         (IF) 02:02:01 1EP  () cdc_acm tty/ttyACM0  
1-1:1.1         (IF) 0a:00:00 2EPs () cdc_acm  
|__1-2         10c4:ea60:0100 00  2.00   12MBit/s 100mA 1IF  (Silicon Labs CP2102N USB to UART Bridge Controller 50678b423885ed11b35dd19f9d1cc348) 
1-2:1.0         (IF) ff:00:00 2EPs ()   
|__1-4         f400:f400:0100 00  2.00  480MBit/s 200mA 1IF  (Synology DiskStation 7F0009292881C640) 
1-4:1.0         (IF) 08:06:50 2EPs () usb-storage host2 (synoboot)
```

**Comment by robertsLando** [comment]
From what I know stick 7 have lot of problems...

cc @AlCalzone any tips?

**Comment by AlCalzone** [comment, marked as answer]
> Silicon Labs CP2102N USB to UART Bridge Controller

is the Z-Stick 7, so it is detected at least. Maybe you're missing a driver? https://github.com/robertklep/dsm7-usb-serial-drivers seems to have some.

Side note though: Your name sounds like you're located in Europe. Z-Stick 7 (like most 700 series controllers) anecdotally work pretty terribly in the EU. I don't know what it is, but several users have tried to make it work and went back to the 500 series after a while.

**Reply by stephanvierkant** [reply]
Yes, I'm in The Netherlands. And I don't know if it works terrible here, because it doesn't work at all. Any idea why it doesn't work on Synology? I installed the drivers.

**Reply by stephanvierkant** [reply]
My bad! I've installed DSM 7.1 drivers, but I'm already on DSM 7.2!
